client/src/thin_client.rs

```diff
@@ -306,11 +307,31 @@ impl SyncClient for ThinClient {
         Ok(self.rpc_client().get_account_data(pubkey).ok())
     }
 
+    fn get_account(&self, pubkey: &Pubkey) -> TransportResult<Option<Account>> {
+        Ok(self.rpc_client().get_account(pubkey).ok())
+    }
+
     fn get_balance(&self, pubkey: &Pubkey) -> TransportResult<u64> {
         let balance = self.rpc_client().get_balance(pubkey)?;
         Ok(balance)
     }
 
+    fn get_recent_blockhash(&self) -> TransportResult<(Hash, FeeCalculator)> {
```

Comment:
CLion complained that the trait implementation was not `in order` of the definition, it automatically moved this fn to match the trait definition